bedrock/firefox/forms.py

```diff
@@ -22,3 +24,21 @@ def clean_number(self):
             number = re.sub(r'\D+', '', number)
 
         return number
+
+
+class UnfckForm(forms.Form):
+    unfck_field = forms.CharField(
+        max_length=500,
```

Comment:
Only 500 characters?

Reply:
Hmm good suggestion, let's increase it to 2000 (does that sound good?)

Reply:
I think it needs to be tweet length.

Reply:
Perhaps we can update this later once the tweet functionality is in place? Right now there's no indication what this form actually does (sending an email or sharing on twitter). I think the UI needs to be made clearer, which can happen in a separate PR.

Reply:
Ok, I've changed this to 280 characters as suggested. I think the UI will really need to make clear the limit to avoid frustration though.

Reply:
Definitely. Probably Twitter-style character countdown.